manifests/admin.pp

```diff
@@ -80,6 +94,8 @@
   Boolean               $force_logged_shell        = true,
   Enum['sudosh','tlog'] $logged_shell              = 'tlog',
   Array[String[2]]      $default_admin_sudo_cmnds  = ['/bin/su - root'],
+  Hash                  $admin_sudo_options        = {},
```

Comment:
We should probably go ahead and drop in the `unconfined_r` transition for the admin users.

We'll need to test and make sure that when SELinux is disabled, there's no issue with having it set but I wouldn't expect there to be.